csvdedupe/csvlink.py

```diff
@@ -18,13 +18,13 @@ def __init__(self):
 
         if len(self.configuration['input']) == 2:
             try:
-                self.input_1 = open(self.configuration['input'][0], 'rU').read()
+                self.input_1 = open(self.configuration['input'][0], encoding='utf-8').read()
```

Comment:
It is already here: https://github.com/dedupeio/csvdedupe/commit/85e55cd84c03bb6e87c0fc355f2571071682c281